api/db.py

```diff
@@ -27,11 +27,16 @@ class Database:
     def __init__(self, host='db', db_name='kernelci'):
         self._motor = motor_asyncio.AsyncIOMotorClient(host=host)
         self._db = self._motor[db_name]
+        self._create_index()
```

Comment:
This will be called every time a `Database` object is created. I think it should instead be called every time the service starts with a `@app.on_event('startup')` function like for starting the Pub / Sub interface.